e2e/babel-plugin-jest-hoist/__tests__/hoistOrder.test.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+'use strict';
+
+import a from '../__test_modules__/a';
+
+// These will all be hoisted above imports
+jest.enableAutomock();
```

Comment:
Can probably remove this e2e test now that there's more direct snapshot tests, will do that after the latest tests complete.